### PR TITLE
Add: support common labels for all resources

### PR DESCRIPTION
**Pull request by thib-mary**
Usefull when we have a policies manager as Kyverno.

Signed-off-by: Flaagada <mary.thibault2@gmail.com>

#### What this PR does / why we need it:

When we used a policies manager as Kyverno, it's usefull to be able to add a common labels on all resources created by the chart in order to respect the policy.

#### Checklist
- [X] Documentation has been updated with helm-docs (run: `.github/helm-docs.sh`)
- [X] Chart Version bumped
- [X] `CHANGELOG.md` has been updated
- [X] Variables are documented in the `README.md`

### CODE REVIEW

File: charts/datadog/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Datadog changelog
 
+## 2.35.4
+
+* Support `commonlabels` configuration to be able to add a common labels on all resources created by the chart
```
Comment (maycmlee) [inline charts/datadog/CHANGELOG.md:5]:
```suggestion
* Support `commonlabels` configuration to be able to add common labels on all resources created by the chart.
```

**Review by maycmlee — COMMENTED** [review]
Small edit!

**Review by maycmlee — APPROVED** [review]
👍 for docs.

**Comment by falzm** [comment]
Hi @maycmlee, is there anything missing to have this PR merged in the main branch? 🙏

**Review by L3n41c — APPROVED** [review]

**Pull request merged**